renku/data/shacl_shape.json

```
@@ -137,10 +141,19 @@
             },
             {
                "path": "schema:creator",
-               "sh:class":{
+               "sh:class": {
                   "@id": "schema:Person"
                },
-               "minCount": 1
+               "minCount": 1,
+               "maxCount": 1,
+               "or": [
+                  {
+                     "sh:pattern": "http(s)?://[^/]+/persons/\\S+"
+                  },
+                  {
+                     "sh:pattern": "mailto:[^@\\s]+\\S+\\.\\S+"
```

```suggestion
                     "sh:pattern": "mailto:[^@\\s]+@\\S+\\.\\S+"
```
I think there is an @ missing in the regex. otherwise `ab.c` would be valid according to this, no?